app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
   before_action :find_user, only: [:destroy, :edit, :update]
+  before_action :admin?
```

Comment:
I'd go with `check_for_admin` or `allow_only_admin` - since it's an action, it should be a verb.,